uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -1011,36 +975,40 @@ define([
        * Sets the min and max for to date from absence period
        */
       function setMinMax() {
-        if (vm.uiOptions.fromDate) {
-          vm.uiOptions.date.to.options.minDate = vm.uiOptions.fromDate;
+        if (this.uiOptions.fromDate) {
```

Comment:
can we rename the function to `setMinMaxDate` to be more descriptive. Its not related but as we are refactoring can we fix this.